nix/sources.json

```diff
@@ -66,15 +66,15 @@
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
     "tezos-packaging": {
-        "branch": "v7.0-1",
+        "branch": "master",
```

Comment:
I guess we can use a tag here again.

Reply:
Though I am not sure it's better than `master` :shrug:

Reply:
Yes, we can use tag now, since 7.4 tag was released